src/ng/compile.js

```diff
@@ -269,12 +269,11 @@
  * #### `scope`
  * The scope property can be `true`, an object or a falsy value:
  *
- * * **falsy:** No scope will be created for the directive. The directive will use its parent's scope.
+ * * **falsy (default):** No scope will be created for the directive. The directive will use its parent's scope.
```

Comment:
If you are adding `(default)` then change it to `false` (because `falsy` is not a concrete value :smiley:).